site/journal/2020/Private Chat and DMs Are Good Actually.md

```diff
@@ -0,0 +1,52 @@
+
+tags:
+  - Slack
+  - chat
+  - productivity
+  - leadership
+
+---
+
+:::note
+
+The topic of public and private chat came up in a group of technical leaders I’m a part of, and I wrote up a note on why I don’t believe that private chats and <abbr title="direct messages">DM</abbr>s are unhealthy—to the contrary. The following is an expanded and edited version of those comments.
+
+:::
+
+It’s common in some circles to treat private rooms or DM-heavy Slack teams as a sign and more importantly a *cause* of dysfunction, particularly in terms of team trust. In particular, folks are sometimes tempted to use the metrics Slack provides to measure the health of their organizations: a high value of the private-to-public messages ratio is seen as a fairly direct measure of mistrust. The corresponding assumption is that it also *causes* teams to silo from each other and mistrust each other in the first place. I disagree firmly with both parts of this, and always have!
+
+First, a heavy use of private rooms and <abbr title="direct messages">DM</abbr>s *might* be a reflection of existing mistrust, but *only* a reflection, not a cause. If you ask around, you’ll find that many organizations with widely varying degrees of siloing, trust, collaboration, etc. have very similar ratios of private and public conversation. (That’s exactly what came out in the conversation that prompted this post, in fact!) If two organizations with polar opposite degrees of trust and collaboration have the same ratio of private-to-public conversation, then siloing and mistrust can’t be caused by private chat. Second, this also means that the ratio of private to public chat isn’t even a reliable signal of whether an organization is healthy or not: it lacks any predictive power!
+
+This is exactly what we should expect based on how people interact with each other in the analog world. We have meeting rooms small and large in offices because having space to talk freely one on one or as a small group helps everyone do better in the public spaces. For one thing, even in the healthiest organization, there are going to be times when you’re frustrated with another person or team. When that happens, the most helpful thing is often to go talk to someone else you trust. That gives you space to blow off steam, to work it through mentally or emotionally so you *can* handle it well publicly.
+
+For another, there are plenty of totally “positive” conversations which benefit from privacy.[^positive] Sometimes you need to backchannel someone to help them avoid tripping over a political or personal landmine in a conversation. Sometimes you’re coaching someone through a difficult personal situation, or through their career considerations. Sometimes you’re working on a thorny and having more input from people with less context would just be a distraction (even if just by way of having to give people enough context to contribute meaningfully).
+
```

Comment:
I'd say, especially in the time of COVID and remote working, people use it for camaraderie and keeping in touch. Like I have a co-worker where we talk about tweaking system configs, tooling, Emacs, and home automation. It's a rolling, random convo between him and I. We chat at least once a day and sometimes a lot. I can't see where leadership would think that this is a bad thing!

Also for answering personal questions (which you kinda covered in "mentoring"). Someone can ask a question in the HR channel about benefits and you have a method on how you handled it, but maybe you don't want that broadcast to the whole organization.

Reply:
Oh, I really like the concrete examples. Thank you!

Reply:
I integrated these into a revision (and further expansion) of the post!